react/features/base/styles/shimStyles.web.js

```diff
@@ -0,0 +1,23 @@
+/**
+ * Shim style properties to work correctly on Web.
```

Comment:
I put the code for web platform from the old repository because `AbstractToolbar` depends on styles module.